pkg/operators/machine-api-operator.go

```diff
@@ -39,6 +39,9 @@ var _ = Describe("[Feature:Operators] Machine API operator deployment should", f
 
 		By(fmt.Sprintf("checking deployment %q is available again", maoManagedDeployment))
 		Expect(framework.IsDeploymentAvailable(client, maoManagedDeployment, framework.MachineAPINamespace)).To(BeTrue())
+
```

Comment:
we should manipulate it, e.g update replica number then wait for it to be reconciled with the enforced spec e.g replica number=1